### ISSUE TITLE
Support remote kind config ref

### ISSUE THREAD

**Issue by nabuskey** [cnoe-io/idpbuilder#454]
Currently the kind config supplied via the `--kind-config` flag can refer to local files only. It should be able to reference remote kind configs as well. This is useful when a stack needs a specific kind configuration. Currently users need to clone the repository to use such stacks. e.g. `--kind-config https://github.com/stacks/kind.config`

**Comment by cmoulliard** [comment]
So your idea is to do this:
- if the string passed to the flag `kind-config` starts with `https://` then we parse the Url
- We download the URL under a temporary folder
- We replace the value of  the field: `NewBuildOptions.KindConfigPath` with the new local path
- We create the kind cluster using the local konfig file

Correct ? @nabuskey

**Comment by nabuskey** [comment]
We probably don't need to download it to a temp folder. I think we can do all that in-memory. Otherwise, yes that's inline with what I was thinking.

**Comment by cmoulliard** [comment]
> I think we can do all that in-memory.

I don't think so as kind go lib expects to read a file - see: https://github.com/kubernetes-sigs/kind/blob/main/pkg/internal/apis/config/encoding/load.go#L31-L51
